apps/web/src/views/TradingReward/components/Leaderboard/MyRank.tsx

```diff
@@ -53,12 +53,12 @@ const MyRank: React.FC<React.PropsWithChildren<MyRankProps>> = ({ campaignId })
                   </Td>
                 </tr>
               ) : (
-                <DesktopResult key={rank.rank} rank={rank} />
+                <DesktopResult key={rank.rank} rank={rank as any} />
               )}
             </tbody>
           </Table>
         ) : (
-          <MobileResult rank={rank} isMyRank />
+          <MobileResult rank={rank as any} isMyRank />
```

Comment:
how come we need any here. could we not just cast the type of rabk on to the component props?

Reply:
if we're using type casting then it doesn't make any difference whether we cast to the correct type or not, since the type is still broken. Will just cast to any to bypass the check first. Clear path for the wagmi migration